misc/acrn-config/schema/config.xsd

```diff
@@ -241,7 +242,7 @@
 <xs:complexType name="VMConfigType">
   <xs:all>
     <xs:element name="vm_type" type="VmOptionsType">
-      <xs:annotation>
+      <xs:annotation acrn:readonly="true">
```

Comment:
It is a boolean value, we follow the original xml format to make the element readonly by acrn:readonly="true".
@szhen11 can you confirm if this would work if it changes from acrn:readonly="y?

Reply:
OK. I will update the UI part following to this change.

Reply:
Thanks, @szhen11.
@dbkinder have made all acrn:readonly="y"